packages/typespec-go/src/lib.ts

```diff
@@ -51,6 +52,11 @@ const EmitterOptionsSchema: JSONSchemaType<GoEmitterOptions> = {
       nullable: true,
       description: 'When true, enables generation of fake servers. The default is false.',
     },
+    'go-generate': {
+      type: 'string',
+      nullable: true,
+      description: `Path to a post-generation 'go generate' script. The path is relative to the directory where the SDK is emitted.`,
+    },
```

Comment:
Is "the directory where the SDK is emitted" the same as the `emitter-output-dir`? If so, it might be good to explicitly use that terminology.

Reply:
Fixed